fbw-a32nx/src/systems/instruments/src/EFB/Store/features/globalSettings.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+// Store for all settings which should be global for the EFB but not neccesary for the simvars.
+
+interface GlobalSettingsState {
+    cabinAnnouncementsActive:boolean,
+    passengerAmbienceActive:boolean,
+    boardingMusicActive:boolean
```

Comment:
```suggestion
    cabinAnnouncementsActive: boolean,
    passengerAmbienceActive: boolean,
    boardingMusicActive: boolean
```

Reply:
Also, isn't there confusion with the method `setPassengerAmbienceEnabled` and `passengerAmbienceActive`?

Reply:
Yes thats right. Active is for the setting the user selects in EFB. Enabled is the name of the simVar. I will change the naming for the setting on something like: cabinAnnouncementSettingEnabled. If you have any idea for a better variable name let me know :)